src/sign-in-xplat/XPlat.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>net6.0-android;net6.0-ios;net6.0-maccatalyst</TargetFrameworks>
+		<TargetFrameworks>net6.0-android;net6.0-maccatalyst</TargetFrameworks>

```

Comment:
it was both i0S and mac we needed removed.

Reply:
you got it @ckittel 

done | addressed from [f1080c](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/18/commits/f1080c85bd3ae08ea847763f6de72aeb201f1760)